driver/src/test/java/oracle/nosql/driver/WriteMultipleTest.java

```diff
@@ -135,7 +136,12 @@ public void testOpSucceed() {
             .setTableName(tableName)
             .setReturnRow(true);
         umRequest.add(put, false);
-        rowPresent.add(true);
+        /* If proxy serial version <= V4,put success does not return row */
```

Comment:
The comment says less than or equal, but the comparison is only less than?

Reply:
Oops. I missed this during refactoring. Fixed now.